roles/servicetelemetry/vars/dummy_user_certs.yml

```diff
@@ -0,0 +1,56 @@
+# These are required because sg-core always expects there to be a user cert + key, whether we need it for auth or not
+# CN = STF DUMMY USER CERT - DO NOT USE
+elastic_user_cert_dummy: |
```

Comment:
Just because I wasn't sure, I looked up if there was a chomp indicator needed here (e.g. `|-` to strip any newlines at the end) and I don't think that's necessary. Noting because I thought of it.